### PR TITLE
client, platform: Use paths.SSH* instead of get_config_dir().

### PR DESCRIPTION
**Pull request by tjaalton**
This cleans things up a bit.

### CODE REVIEW

**Comment by tjaalton** [comment]
I don't understand why travis claims it failed while the build passed fine

**Comment by mirielka** [comment]
Build passes fine, pep8 does not:
./ipaclient/install/client.py:1006:80: E501 line too long (82 > 79 characters)
./ipaclient/install/client.py:1029:80: E501 line too long (83 > 79 characters)

**Comment by tiran** [comment]
@tjaalton Travis CI output is a bit confusing. PEP 8 checks are performed first. The remaining checks are still executed.

@martbab What do you think about improving the output to make it friendlier to outside contributors? Either fail fast or repeat PEP 8 errors at the end of the log.

**Comment by tjaalton** [comment]
oh, thanks for pointing that out.. new one pushed

**Comment by martbab** [comment]
@tiran I agree, PEP8 errors are often overlooked in the slew of other stuff that gets printed out in the container. I will address this in https://github.com/freeipa/freeipa/pull/293

File: ipaplatform/base/paths.py [hunk 1]
```diff
@@ -102,6 +102,7 @@ class BasePathNamespace(object):
     SMB_CONF = "/etc/samba/smb.conf"
     LIMITS_CONF = "/etc/security/limits.conf"
     SSH_CONFIG = "/etc/ssh/ssh_config"
+    SSH_CONFIG_DIR = "/etc/ssh"
```
Comment (mbasti-rh) [inline ipaplatform/base/paths.py]: Can you please move it one line up, list of path is sorted alphabetically by path, not by variable name

**Comment by mbasti-rh** [comment]
@tjaalton just one nitpick, otherwise it looks good to me

**Comment by mbasti-rh** [comment]
Fixed upstream
master:
https://fedorahosted.org/freeipa/changeset/0ff12de338a8db32bb10e1b41f32255e7b971b6f

**Pull request closed without merging**